src/query/expression/src/values.rs

```diff
@@ -1185,6 +1207,16 @@ impl Column {
                 }
                 VariantType::from_data(data)
             }
+            DataType::Geometry => {
+                let mut data = Vec::with_capacity(len);
+                (0..len).for_each(|_| {
+                    let x = SmallRng::from_entropy().gen::<f64>();
+                    let y = SmallRng::from_entropy().gen::<f64>();
+                    let val = Point::new(x, y);
+                    data.push(serde_json::to_vec(&val).unwrap());
```

Comment:
geo has a project working on the implementation of parquet: https://geoparquet.org/
Many open source projects are compatible with the geoparquet, such as [apache sedona](https://sedona.apache.org/latest/).

Based on geo-parquet, parquet can do special metadata and indexes for geo types, and stored in wkb format.

Can we use this project and related projects directly to do the implementation, and then do separate indexing and optimization for the geo type behind the aspect?